benchmarks/README.md

```diff
@@ -0,0 +1,13 @@
+# Caikit NLP Tuning Benchmarks
+
+Benchmark results for various model on various hardware configurations.
+
+## Llama2-7b
+
+| Date Executed |   Hardware   | Training Set | Epoch | Precision | Batch Size | Max Source Length | Training Runtime (s) | Samples Per Second | Train Steps Per Second | Loss |        Notes         |
+|---|---|---------------|---|---|:---:|---|------------| --- |---|---|---|
+| [2023-09-05](./logs/llama2-7b/20230905_183655.output) | 1 x A100 80GB | [Glue / RTE](https://huggingface.co/datasets/glue) | 1 | bfloat16 | 6 | 4096 | 350 | 21.325 | 0.22 | 1.65 |     4096 is the context size for Llama2     |
+| [2023-09-05](./logs/llama2-7b/20230905_184809.output) | 1 x A100 80GB | [Glue / RTE](https://huggingface.co/datasets/glue) | 1 | bfloat16 | 6 | 1024 | 350 | 21.333 | 0.22 | 1.65 | batch size of 7 fails CUDA OOM |
```

Comment:
I think RTE might have text long enough for 4096 context length, but definitely good to note and then we can iterate later on with a different dataset..